src/db/tripledb.plt

```diff
@@ -1,54 +1,69 @@
 :- begin_tests('tripledb').
```

Comment:
I think it would be good to define a *cleanup* goal here where you delete all triples from the graph names "test".
In addition, a setup where you set the default graph to "test". Then all *tell*'s will go into the "test" graph.

Finally, you can then also call *tripledb_load* and load the triples into "test" graph.